src/rss2.ts

```diff
@@ -157,6 +157,7 @@ export default (ins: Feed) => {
     if (Array.isArray(entry.author)) {
       item.author = [];
       entry.author.map((author: Author) => {
+        item["dc:creator"] = { _cdata: author.name };
         if (author.email && author.name) {
           item.author.push({ _text: author.email + " (" + author.name + ")" });
         }
```

Comment:
```suggestion
      item["dc:creator"] = [];
      entry.author.map((author: Author) => {
        if (author.email && author.name) {
          item.author.push({ _text: author.email + " (" + author.name + ")" });
        } else if (author.name) {
          isDublinCore = true;
          item["dc:creator"].push({ _cdata: author.name });
        }
```

And up above:
```
  let isDublinCore = false;
```

And down below:

```
  if (isDublinCore) {
    base.rss._attributes["xmlns:dc"] = "http://purl.org/dc/elements/1.1/";
  }
```